patterns/footer.php

```diff
@@ -7,81 +7,80 @@
-<div class="wp-block-group alignfull has-primary-color has-background-background-color has-text-color has-background" style="padding-top:20px;padding-bottom:20px"><!-- wp:spacer {"height":"10px"} -->
-<div style="height:10px" aria-hidden="true" class="wp-block-spacer"></div>
-<!-- /wp:spacer -->
-
-<!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}},"border":{"top":{"width":"0px","style":"none"},"right":{"width":"0px","style":"none"},"bottom":{"width":"0px","style":"none"},"left":{"color":"var:preset|color|primary","width":"1px"}}},"layout":{"type":"flex","flexWrap":"wrap","verticalAlignment":"top"}} -->
-<div class="wp-block-group" style="border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-color:var(--wp--preset--color--primary);border-left-width:1px;padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}}} -->
-<h3 style="text-transform:uppercase">pages</h3>
-<!-- /wp:heading -->
-
-<!-- wp:group {"style":{"border":{"width":"0px","style":"none"}},"layout":{"type":"flex","flexWrap":"nowrap"},"fontFamily":"eb-garamond"} -->
-<div class="wp-block-group has-eb-garamond-font-family" style="border-style:none;border-width:0px"><!-- wp:paragraph {"fontSize":"small"} -->
-<p class="has-small-font-size">About</p>
-<!-- /wp:paragraph -->
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"80px","bottom":"0px","right":"20px","left":"20px"}},"border":{"top":{"width":"1px"}}},"backgroundColor":"background","textColor":"primary","layout":{"type":"constrained","contentSize":"1000px"}} -->
+<div class="wp-block-group alignfull has-primary-color has-background-background-color has-text-color has-background" style="border-top-width:1px;padding-top:80px;padding-right:20px;padding-bottom:0px;padding-left:20px"><!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"40px","left":"40px"}}}} -->
+<div class="wp-block-columns"><!-- wp:column {"width":"81.7%","style":{"typography":{"lineHeight":"1.3"}}} -->
+<div class="wp-block-column" style="line-height:1.3;flex-basis:81.7%"><!-- wp:columns {"style":{"border":{"left":{"width":"1px"}},"spacing":{"padding":{"left":"20px"},"blockGap":{"top":"10px","left":"20px"}}},"className":"course-footer-links-vertical-space"} -->
+<div class="wp-block-columns course-footer-links-vertical-space" style="border-left-width:1px;padding-left:20px"><!-- wp:column {"verticalAlignment":"center","width":"148px"} -->
+<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:148px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase","lineHeight":"1"}},"className":"has-gothic-font-family","fontSize":"medium"} -->
+<h3 class="has-gothic-font-family has-medium-font-size" style="line-height:1;text-transform:uppercase">social</h3>
```

Comment:
We need to ensure we're internationalizing all strings so they can be translated.

Reply:
Thanks @donnapep, updated [here](https://github.com/Automattic/sensei-theme/pull/70/commits/15b53b9b486c757e63bc948236d7f1f16ff69bae)

Reply:
@donnapep I reviewed the changes requested and it looks good so I'm gonna approve

Reply:
@LavaGolem I didn't do a full review of this, just noticed the strings. Did you review the entire PR?